### DISCUSSION TITLE
Podcast listening progress tracking, server-side.

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#15465] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

Nope!

**Describe the solution you'd like**

I would to ensure that whenever I am listening to a Podcast, on any device, I won't lose progress either when iOS tries to reclaim resources from an idle app or when switching between web and mobile. The podcast listening bar is always persistent when I login from any device.

**Describe alternatives you've considered**

Trying to memorize the progress so when it disappears on iOS, I can remember where I am up to again.

**Additional context**

I think this one feature is one of the final steps to this becoming a good Podcast replacement on mobile.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by cmgorton** [comment]
Since this issue has gone stale I am going to close it. If anyone would like to still implement this feature let me know and we can turn this issue into a [discussion](https://github.com/forem/forem/discussions)
